virtualization/Docker/Dockerfile.dev

```diff
@@ -0,0 +1,55 @@
+# Dockerfile for development
+# Based on the production Dockerfile, but with development additions.
+# Keep this file as close as possible to the production Dockerfile, so the environments match.
+
+FROM python:3.5
```

Comment:
I would love to do something like that, but that would add a requirement on the Dockerfile on dockerhub. This would make it impossible to change anything in the Dockerfile on your local development branch, since the image from dockerhub would be fetched every time. This doesn't seem like a good idea.

A better approach would be to depend on the main Dockerfile, but docker doesn't want to implement this (see https://github.com/docker/docker/issues/735).